query_optimizer/ExecutionGenerator.cpp

```diff
@@ -135,7 +135,7 @@ static const volatile bool join_hashtable_type_dummy
     = gflags::RegisterFlagValidator(&FLAGS_join_hashtable_type,
                                     &ValidateHashTableImplTypeString);
 
-DEFINE_string(aggregate_hashtable_type, "LinearOpenAddressing",
+DEFINE_string(aggregate_hashtable_type, "SeparateChaining",
```

Comment:
@jianqiao I understand that the performance of a particular query (Q18) is poor when using the default `LinearOpenAddressing`, but I was wondering if we do need to change the default value.

In general, `LinearOpenAddressing` would have a better performance over `SeparateChaining` version, as we avoid a look up in a long chain of hashed keys.

Alternatively, for some slow queries, we could just set a different hash table implementation to `SeparateChaining` instead of using the default option.